bson/bsoncodec/struct_codec.go

```diff
@@ -257,6 +283,23 @@ func (sc *StructCodec) isZero(i interface{}) bool {
 		return v.Float() == 0
 	case reflect.Interface, reflect.Ptr:
 		return v.IsNil()
+	case reflect.Struct:
+		if sc.OmitDefaultStruct {
```

Comment:
Does having this here change the current behavior? It seems like if we don't enter this, we are always returning false.

I guess by the naming of this, that is inevitable, but it's unclear as to whether current users are wanting this.

Reply:
Originally isZero always returned false for structs that don't implement the Zeroer interface, so the default behavior is the same.

Reply:
works for me. :)